.github/workflows/perl-versions.yml

```diff
@@ -0,0 +1,27 @@
+name: List perl versions
```

Comment:
should we consider running that workflow on PR for sanity check?

Reply:
I'm thinking about that so far no idea (but I'm sure I will come with some)
so far I have (private) repo where I call workflows with manual dispatch.